src/Vote.Monitor.Api.Feature.PollingStation.Attachments/Specifications/GetMonitoringObserverSpecification.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Ardalis.Specification;
+using Vote.Monitor.Domain.Entities.MonitoringObserverAggregate;
+
+namespace Vote.Monitor.Api.Feature.PollingStation.Attachments.Specifications;
+public sealed class GetMonitoringObserverSpecification : Specification<MonitoringObserver>
```

Comment:
we expect a single result when running this specification so it should extend `SingleResultSpecification<MonitoringObserver>`